lightning/src/onion_message/packet.rs

```diff
@@ -136,6 +136,19 @@ impl<T: CustomOnionMessageContents> OnionMessageContents<T> {
 	}
 }
 
+impl<T: CustomOnionMessageContents> fmt::Display for OnionMessageContents<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            OnionMessageContents::Offers(msg) => {
+                write!(f, " (Offers): {:?}", msg)
```

Comment:
This writes the `Debug` of the `msg`, but that uses the derived `Debug` instead of your newly added code.